frontend/src/metabase/visualizations/components/MiniBar.jsx

```diff
@@ -81,7 +81,7 @@ const MiniBar = ({ value, extent: [min, max], options }) => {
               left: "50%",
               top: 0,
               bottom: 0,
-              borderLeft: `1px solid white`,
+              borderLeft: "1px solid var(--mb-color-white)",
```

Comment:
I don't think we have `--mb-color-white` yet.

Reply:
I guess you should use `color("white")` instead.

Reply:
Nice catch!